react/features/reactions/components/web/ReactionsMenuButton.tsx

```diff
@@ -60,13 +69,28 @@ interface IProps extends WithTranslation {
     reactionsQueue: Array<IReactionEmojiProps>;
 }
 
+
+/**
+ * Implementation of a button for reactions.
+ */
+class ReactionsButtonImpl extends AbstractButton<AbstractButtonProps> {
```

Comment:
I'm not a fan of this naming. Please leave it as `ReactionsButton` and extract the other component (`ReactionsMenuButton`) to another file

Reply:
I'm not a fan either. Now the problem is that we already have a file and component called `ReactionsButton`!

Also `ReactionsButtonImpl` will ever be needed only here. 

I was thinking about a better solution but failed to find one. If you have better idea please let me know!